tools/releaseBuild/azureDevOps/templates/release-PublishPackageMsftCom.yml

```diff
@@ -0,0 +1,22 @@
+steps:
+- template: release-SetReleaseTagAndContainerName.yml
+
+- pwsh: |
+    $packageVersion = '$(ReleaseTag)'.ToLowerInvariant() -replace '^v',''
+    $vstsCommandString = "vso[task.setvariable variable=packageVersion]$packageVersion"
+    Write-Host "sending " + $vstsCommandString
+    Write-Host "##$vstsCommandString"
+  displayName: Set Package version
+
+- pwsh: |
+   git clone https://$(AzureDevOpsPat)@mscodehub.visualstudio.com/PowerShellCore/_git/Internal-PowerShellTeam-Tools '$(Pipeline.Workspace)/tools'
+  displayName: Clone Internal-PowerShellTeam-Tools from MSCodeHub
+
+- download: releasePipeline
+  artifact: metadata
+
+- pwsh: |
+    $metadata = Get-Content -Path "$ENV:PIPELINE_WORKSPACE/releasePipeline/metadata/release.json" -Raw | ConvertFrom-Json
+    Write-Verbose -Verbose "$(releaseTag) $(PackagesRepoPassword) $(AzureVersion) $($metadata.LTSRelease)"
```

Comment:
declare what variable group we depend on

Reply:
fixed